pkg/maintenance/postgresql.go

```diff
@@ -102,15 +120,29 @@ func (p *PostgreSQL) DoMaintenance(ctx context.Context) error {
 	}
 
 	p.log.Info("Repacking databases...")
-	err = p.createRepack(clusterName)
+	err = p.createRepack(sgCluster.GetName())
 	if err != nil {
 		return fmt.Errorf("cannot create repack: %v", err)
 	}
 
 	return nil
 }
 
-func (p *PostgreSQL) upgradeVersion(currentVersion string, clusterName string) (OpName, error) {
+func (p *PostgreSQL) checkRestart(sgCluster stackgresv1.SGCluster) (bool, error) {
```

Comment:
I don't understand why this function is call `checkRestart()`

Reply:
No it should make sense (see above thread)